superbuild/projects_modules/LogDemons.cmake

```diff
@@ -39,8 +39,8 @@ if (NOT USE_SYSTEM_${ep})
 ## Set up versioning control.
 ## #############################################################################
 
-set(git_url ${GITHUB_PREFIX}Inria-Asclepios/LCC-LogDemons.git)
-set(git_tag master)
+set(git_url ${GITHUB_PREFIX}medInria/LCC-LogDemons.git)
```

Comment:
Will need to be changed back to Inria-Asclepios master once all branches are validated and merged